tests/View/ViewBladeCompilerTest.php

```diff
@@ -207,6 +207,30 @@ public function testCommentsAreCompiled()
 this is a comment
 */ ?>';
         $this->assertEquals($expected, $compiler->compileString($string));
+
+        $string = '{{--
+this is an extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely
+extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely
+extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely
+extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely
+extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely
+extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely
+extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely
+extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely
+extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely long comment
+--}}';
+        $expected = '<?php /*
+this is an extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely
+extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely
+extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely
+extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely
+extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely
+extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely
+extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely
+extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely
+extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely long comment
+*/ ?>';
+        $this->assertEquals($expected, $compiler->compileString($string));
```

Comment:
@vlakoff Yeah good thinking, test looks terrible, haha...

@JosephSilber That sounds like a pretty good idea to me, hella easy too...